### PR TITLE
docs(content/mig-guide): migration to HeroDevs XLTS libraries

### PR DESCRIPTION
**Pull request by venetrius**
related to: https://github.com/camunda/camunda-bpm-platform/issues/4549

### CODE REVIEW

File: content/update/patch-level.md [hunk 1]
```diff
@@ -538,6 +538,16 @@ Starting with version 7.22.1, the behavior has changed: if any of the process in
 
 Please note that this does not apply to the Sync API, which keeps its behavior and fails if the process instance does not exist.
 
+## 7.22.0 to 7.22.1 / 7.21.5 to 7.21.6 /  7.20.8 to 7.20.9
```
Comment (venetrius) [inline content/update/patch-level.md:541]: We already had a section for 
```
## 7.22.0 to 7.22.1
```
I created a new section as this is relevant also for migrating: `7.21.5 to 7.21.6 /  7.20.8 to 7.20.9`
Let me know if there is a better way to do this.

**Comment by venetrius** [comment]
This change is blocked, moved into draft.

File: content/introduction/third-party-libraries/_index.md [hunk 2]
```diff
@@ -10,18 +10,23 @@ menu:
...
-
-For selected third-party libraries that we consider especially noteworthy, this page describes their license terms and use in the Camunda 7 artifacts.
+This section covers third party libraries and their use in Camunda. It used to provides license books that list the third-party dependencies that Camunda distributes along with its code. Starting with versions 7.20.9, 7.21.6, 7.22.1 all licenses are included in our license book.
 
 For legal reference and any other license-related aspects please refer to [Licences]({{< ref "/introduction/licenses.md" >}}).
 
 
 # Web Applications (Cockpit, Tasklist, Admin)
 
 ## XLTS for AngularJS
```
Comment (venetrius) [inline content/introduction/third-party-libraries/_index.md:20]: This section is referenced by old migration guides so I opted to only edit it, but not remove.

File: content/update/patch-level.md [hunk 3]
```diff
@@ -538,6 +538,17 @@ Starting with version 7.22.1, the behavior has changed: if any of the process in
 
 Please note that this does not apply to the Sync API, which keeps its behavior and fails if the process instance does not exist.
 
+### XLTS libraries provided by XLTS.dev by HeroDevs, Incl
+
+Starting with 7.22.1,  7.21.6, 7.20.9 the following libraries are replaced with XLTS versions provided by XLTS.dev by HeroDevs, Incl:
```
Comment (tasso94) [inline content/update/patch-level.md:543]:
```suggestion
These patches replace Bootstrap NES and AngularJS NES versions provided by HeroDevs, Inc.:
```

File: content/update/patch-level.md [hunk 4]
```diff
@@ -538,6 +538,17 @@ Starting with version 7.22.1, the behavior has changed: if any of the process in
...
+
+*  *angular-ui-bootstrap*
```
Comment (tasso94) [inline content/update/patch-level.md:545]:
```suggestion
* *AngularJS* (technical names: `angular`, `angular-animate`, `angular-cookies`, `angular-loader`, `angular-mocks`, `angular-resource`, `angular-route`, `angular-sanitize`, `angular-touch`)
*  *angular-ui-bootstrap*
```

File: content/update/patch-level.md [hunk 5]
```diff
@@ -538,6 +538,17 @@ Starting with version 7.22.1, the behavior has changed: if any of the process in
...
+*  *angular-translate*
+*  *angular-moment*
+*  *Bootstrap*
```
Comment (tasso94) [inline content/update/patch-level.md:548]:
```suggestion
*  *Bootstrap*

Where AngularJS, angular-ui-bootstrap, angular-translate, angular-moment, and Bootstrap were licensed entirely under the MIT license, Bootstrap NES and AngularJS NES by HeroDevs, Inc. licenses additional parts under the HeroDevs NES License. By downloading and using Camunda with Bootstrap NES and AngularJS NES by HeroDevs, Inc., you agree to the terms of the HeroDevs NES License. You can find the HeroDevs NES License terms in our [License Book](TODO: ADD LINK TO LICENSE BOOK).
```

File: content/update/patch-level.md [hunk 6]
```diff
@@ -538,6 +538,17 @@ Starting with version 7.22.1, the behavior has changed: if any of the process in
...
+
+Please see our [third-Party libraries documentation]({{< ref "/introduction/third-party-libraries/_index.md#web-applications-cockpit-tasklist-admin" >}}) for details.
```
Comment (tasso94) [inline content/update/patch-level.md:550]:
```suggestion
Please see our [third-Party libraries documentation]({{< ref "/introduction/third-party-libraries/_index.md#web-applications-cockpit-tasklist-admin" >}}) for details.
```

**Review by tasso94 — CHANGES_REQUESTED** [review]

File: content/introduction/third-party-libraries/_index.md [hunk 7]
```diff
@@ -20,8 +20,12 @@ For legal reference and any other license-related aspects please refer to [Licen
...
-
-Starting with versions 7.18.0-alpha2, 7.17.2, 7.16.9, 7.15.15, the Camunda web applications use a set of third-party libraries referred to as *XLTS for AngularJS* (technical names: `angular`, `angular-animate`, `angular-cookies`, `angular-loader`, `angular-mocks`, `angular-resource`, `angular-route`, `angular-sanitize`, `angular-touch`). XLTS for AngularJS follows a proprietary license called *EULA for the downstream recipient of XLTS for AngularJS* (short *XLTS for AngularJS – EULA*) that you can find [here](https://xlts.dev/angularjs/downstream-eula).
+Starting with versions 7.20.9, 7.21.6, 7.22.1 the Camunda web applications use a set of third-party libraries through XLTS.dev by HeroDevs, Incl
```
[the remark this answers is not in the document]
Comment (venetrius) [reply content/introduction/third-party-libraries/_index.md]: Applied suggestions and they updated all cases.

File: content/introduction/third-party-libraries/_index.md [hunk 8]
```diff
@@ -10,18 +10,25 @@ menu:
...
 
 # Web Applications (Cockpit, Tasklist, Admin)
 
 ## XLTS for AngularJS
+Starting with versions 7.18.0-alpha2, 7.17.2, 7.16.9, 7.15.15 and up to including 7.20.8, 7.21.5, 7.22.0, and 7.23.0-alpha1 the Camunda web applications used a set of third-party libraries provided by XLTS.dev
+
+You can find our rationale for using long-term support libraries in [our blog post on ensuring the long-term maintenance of Camunda 7](https://camunda.com/blog/2022/02/ensuring-continuous-support-of-angularjs-in-camunda-platform-7-17/).
 
-Starting with versions 7.18.0-alpha2, 7.17.2, 7.16.9, 7.15.15, the Camunda web applications use a set of third-party libraries referred to as *XLTS for AngularJS* (technical names: `angular`, `angular-animate`, `angular-cookies`, `angular-loader`, `angular-mocks`, `angular-resource`, `angular-route`, `angular-sanitize`, `angular-touch`). XLTS for AngularJS follows a proprietary license called *EULA for the downstream recipient of XLTS for AngularJS* (short *XLTS for AngularJS – EULA*) that you can find [here](https://xlts.dev/angularjs/downstream-eula).
+### Bootstrap NES and AngularJS NES by HeroDevs, Inc.
```
Comment (tasso94) [inline content/introduction/third-party-libraries/_index.md:25]:
```suggestion
## Bootstrap NES and AngularJS NES by HeroDevs, Inc.
```

File: content/update/patch-level.md [hunk 9]
```diff
@@ -538,6 +538,20 @@ Starting with version 7.22.1, the behavior has changed: if any of the process in
 
 Please note that this does not apply to the Sync API, which keeps its behavior and fails if the process instance does not exist.
 
+### Bootstrap NES and AngularJS NES by HeroDevs, Inc.
+
+These patches replace Bootstrap NES and AngularJS NES versions provided by HeroDevs, Inc.:
```
Comment (tasso94) [inline content/update/patch-level.md:543]:
```suggestion
These patches replace the following libraries with versions of Bootstrap NES and AngularJS NES by HeroDevs, Inc.:
```

**Review by tasso94 — CHANGES_REQUESTED** [review]
Looks almost ready to me. One more thing:
❓ Should we add what we have in the patch level guide with a few adjustments to the minor update guide under `/update/minor/722-to-723/`?

File: content/introduction/third-party-libraries/_index.md (hunk 8 above)
Comment (venetrius) [reply content/introduction/third-party-libraries/_index.md]: Added to local commit.

**Comment by venetrius** [comment]
> Looks almost ready to me. One more thing: ❓ Should we add what we have in the patch level guide with a few adjustments to the minor update guide under `/update/minor/722-to-723/`?

Good point, added update to minor update guide under `/update/minor/722-to-723/`

File: content/update/minor/722-to-723/_index.md [hunk 10]
```diff
@@ -25,4 +26,18 @@ Before version 7.22.1, the Set Variables Async API failed whenever at least one
 
 Starting with version 7.22.1, the behavior has changed: if any of the process instances was deleted or completed, the call will still succeed. As a consequence, the Set Variables Batch Operation will also succeed in this case.
 
-Please note that this does not apply to the Sync API, which keeps its behavior and fails if the process instance does not exist.
\ No newline at end of file
+Please note that this does not apply to the Sync API, which keeps its behavior and fails if the process instance does not exist.
+
+# Bootstrap NES and AngularJS NES by HeroDevs, Inc.
+
+Camunda 7.23.1 replaces the following libraries with versions of Bootstrap NES and AngularJS NES by HeroDevs, Inc.:
```
Comment (tasso94) [inline content/update/minor/722-to-723/_index.md:33]:
```suggestion
Camunda 7.23.0 replaces the following libraries with versions of Bootstrap NES and AngularJS NES by HeroDevs, Inc.:
```

**Review by tasso94 — APPROVED** [review]
Looks almost perfect now! Just a tiny typo in the new migration guide addition. You can merge without further review once fixed.

**Pull request merged**